### ISSUE TITLE
[Bug] gt_bboxes_ignore is not being used is almost all models

### ISSUE THREAD

**Issue by levan92** [open-mmlab/mmdetection#10019]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmdetection

### Environment

sys.platform: linux
Python: 3.8.8 (default, Apr 13 2021, 19:58:26) [GCC 7.3.0]
CUDA available: True
GPU 0,1,2,3: Tesla V100-PCIE-16GB
CUDA_HOME: /usr/local/cuda
NVCC: Cuda compilation tools, release 11.1, V11.1.105
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
PyTorch: 1.9.0
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) oneAPI Math Kernel Library Version 2021.2-Product Build 20210312 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.1.2 (Git Hash 98be7e8afa711dc9b66c8ff3504129cb82013cdb)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_61,code=sm_61;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=compute_37
  - CuDNN 8.0.5
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.1, CUDNN_VERSION=8.0.5, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow
-Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.9.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON,

TorchVision: 0.10.0
OpenCV: 4.5.3
MMCV: 1.6.0
MMCV Compiler: GCC 7.5
MMCV CUDA Compiler: 11.1
MMDetection: 2.25.0+bacbb47

### Reproduces the problem - code sample

`all configs`

### Reproduces the problem - command or script

`all configs`

### Reproduces the problem - error message

`all configs`

### Additional information

The primary purpose of loading `gt_bboxes_ignore` during the dataset pipelines is so that we can ignore loss computation in those regions of ignore/crowd. However, I see that all the configs in mmdetection do not "collect" `gt_bboxes_ignore` in the `Collect` step of the data pipelines. Is `gt_bboxes_ignore` usually ignored?

**Comment by hhaAndroid** [comment]
@levan92 Hi. The mmdet 3.x branch already supports ignore data processing. but most models still don't take this data into account. YOLOv5 in MMYOLO has already implemented ignore processing, you can refer to https://github.com/open-mmlab/mmyolo/blob/main/configs/yolov5/README.md#crowdhuman

**Comment by levan92** [comment]
Thanks for your prompt reply! It does seems like in the mmyolo implementation of YOLOv5, they have taken account to ignore the grids associated to the ignore regions. 

It is interesting to note that we have been ignoring the "crowd" and "ignore" attribute labels all these while, this is not a dataset specific problem, in fact in COCO, there are already images with "iscrowd" boxes. For example (grey bbox is labelled with `iscrowd`): 

<img width="543" alt="image" src="https://user-images.githubusercontent.com/13159336/227452450-923e73a9-2aaf-4811-98d3-230315e1af44.png">

From your understanding, does this mean that all these while, during training, if the model predicts a person instance in the "crowd" region and it does match with any GT labels (because it is already covered by the "iscrowd" box), the model is still getting penalised (adds to its loss) for this false "false positive"? Is it the case that such occurrences are not too detrimental to model training?

**Comment by Renat2001** [comment]
Good day!

@levan92 Have you resolved your issue with `gt_bboxes_ignore`? Are there any other models than YOLOv5 that correctly implemented ignoring mechanism (Specifically, Instance Segmentation)?
